lib/reviewlette.rb

```diff
@@ -5,6 +5,10 @@
 
 VERSION = '0.0.10'
 
+# Assume cards have following card title when estimated
+# (8) This is the card name'
+POINTS_REGEX = /\(([\d.]+)\)/
```

Comment:
Why do you have a dot in there? Do we support non-integer amounts of points?
Maybe use `\d+(.\d+)?` then?
Otherwise something like `2.8.5` would match/